### PR TITLE
Swift protocol

### PR DESCRIPTION
**Pull request by manishdadwal13**
Please review the swift protocol and I have implemented combine with it

### CODE REVIEW

File: ingine/ingine/Additional View Controllers/StatusViewController.swift [hunk 1]
```diff
@@ -148,23 +150,23 @@ extension StatusViewController {
...
-    
-        showMessage("\(message)\n\(Double((Double(startIndex)/Double(endIndex))*100).rounded(toPlaces: 0))% Complete", autoHide: false)
+//        guard let dict = notification.userInfo,
+//            let message = dict[NotificationProgressUserInfoType.Message.rawValue] as? String,
+//            let startIndex = dict[NotificationProgressUserInfoType.StartingIndex.rawValue] as? Int,
+//            let endIndex = dict[NotificationProgressUserInfoType.EndingIndex.rawValue] as? Int else {
+//                return
+//        }
+//
+//        showMessage("\(message)\n\(Double((Double(startIndex)/Double(endIndex))*100).rounded(toPlaces: 0))% Complete", autoHide: false)
```
Comment (jtcreative) [inline ingine/ingine/Additional View Controllers/StatusViewController.swift:160]: Why is this commented out

Comment (jtcreative) [inline ingine/ingine/Additional View Controllers/StatusViewController.swift:160]: Why is this commented out?

**Review by jtcreative — COMMENTED** [review]
These changes are not hitting the mark at all. 

- The PR contains different code. There is also the code to upload photos
- There is a lot of commented code
- The primary objective was not achieved

I am concerned about the code quality and the example I gave you have deviated from. In light of time constraints, I will merge this into the codebase.

File: ingine/ingine/Additional View Controllers/StatusViewController.swift (hunk 1 above)
Comment (manishdadwal13) [reply ingine/ingine/Additional View Controllers/StatusViewController.swift:160]: I have created another class i.e. Notification Data Binging for this tasks that's why I commented this lines

Comment (manishdadwal13) [reply ingine/ingine/Additional View Controllers/StatusViewController.swift:160]: I have created another class i.e. Notification Data Binging for this tasks that's why I commented this lines

**Pull request merged**